components/script/dom/html/htmlmediaelement.rs

```diff
@@ -3074,7 +3074,14 @@ impl VirtualMethods for HTMLMediaElement {
 
         match *attr.local_name() {
             local_name!("muted") => {
-                self.SetMuted(mutation.new_value(attr).is_some());
+                if let AttributeMutation::Set(_, reason) = mutation {
```

Comment:
I would have thought that the following also works:


```suggestion
                if let AttributeMutation::Set(_, AttributeMutationReason::ByCloning | AttributeMutationReason::ByParser) = mutation {
```

Not sure though and also the current code is also fine by me. Just something to tinker with when destructuring.